airbyte-webapp/src/components/ui/Text/Text.tsx

```diff
@@ -42,29 +43,36 @@ const getTextClassNames = ({
   });
 };
 
-export const Text: React.FC<React.PropsWithChildren<TextProps>> = React.memo(
-  ({
-    as = "p",
-    bold = false,
-    centered = false,
-    children,
-    className: classNameProp,
-    size = "md",
-    inverseColor = false,
-    color = "default",
-    gradient = false,
-    ...remainingProps
-  }) => {
-    const className = classNames(
-      getTextClassNames({ centered, size, bold, inverseColor, gradient, color }),
-      classNameProp
-    );
+export const Text: React.FC<React.PropsWithRef<React.PropsWithChildren<TextProps>>> = React.memo(
+  React.forwardRef(
```

Comment:
Ugly diff -- this was wrapped in a forwardRef and the ref was passed down to the text